src/platform/logging/BUILD.gn

```diff
@@ -0,0 +1,84 @@
+# See Project CHIP LICENSE file for licensing information.
+
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+import("//build_overrides/pigweed.gni")
+
+import("${chip_root}/src/platform/device.gni")
+
+config("includes") {
+  include_dirs = [ "include" ]
+}
+
+if (chip_enable_openthread) {
+  import("//build_overrides/openthread.gni")
+
+  if (chip_device_platform == "linux" || chip_device_platform == "Darwin") {
+    import("//build_overrides/ot_br_posix.gni")
+  }
+}
+
+simple_log_platforms = [
+  "cc13x2_26x2",
+  "darwin",
+  "efr32",
+  "esp32",
+  "k32w",
+  "linux",
+  "nrfconnect",
+  "qpg6100",
+]
+
+foreach(platform, simple_log_platforms) {
+  if (chip_device_platform == platform) {
+    static_library(platform) {
+      sources = [ "impl/" + platform + "/Logging.cpp" ]
+
+      deps = [
+        "${chip_root}/src/lib/core:chip_config_header",
+        "${chip_root}/src/platform:platform_buildconfig",
+      ]
+
+      public_configs = [ ":includes" ]
```

Comment:
Can we avoid adding this and instead include relative to src/ ?

Reply:
Changed a bit. I am not sure what you mean by relative to src ... please take a look if what I changed makes sense.

Reply:
e.g. src/controller/CHIPDeviceController.h is included by

`#include <controller/CHIPDeviceController.h>`

So my request is to include this as

`#include <platform/logging/include/LogV.h>`

but maybe drop the "include" directory and just have

`#include <platform/logging/LogV.h>`

(include style is inconsistent between src/, /src/lib/, and src/platform - we should fix it)

Reply:
seems reasonable - Updated.